Aspire BudgetingTests/StateManagerTests.swift

```diff
@@ -0,0 +1,58 @@
+//
+//  StateManagerTests.swift
+//  Aspire BudgetingTests
+//
+//  Created by TeraMo Labs on 11/16/19.
+//  Copyright © 2019 TeraMo Labs. All rights reserved.
+//
+
+@testable import Aspire_Budgeting
+import XCTest
+
+class StateManagerTests: XCTestCase {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/aspirebudgetingmobile/aspirebudgeting_ios/blob/d4b9b8c66d27490529e3aeb230968f661ecd4f82/Aspire BudgetingTests/StateManagerTests.swift#L12"></span>Consider using final for this class or use a struct (final_class)</td>
    </tr>
  </tbody>
</table>